examples/js/effects/VREffect.js

```diff
@@ -61,25 +61,51 @@ THREE.VREffect = function ( renderer, onError ) {
 
 	this.scale = 1;
 
+	this.bufferScale = 1.0;
```

Comment:
I'm not sure this is needed. Duplicates `renderer.setPixelRatio()` functionality.